src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -45,6 +47,8 @@ public class MainWindow extends UiPart<Stage> {
     @FXML
     private StackPane personListPanelPlaceholder;
 
+    @FXML
+    private StackPane scheduleListPanelPlaceholder;
     @FXML
```

Comment:
Should there be an empty line between the 2 FXML objects?

Reply:
I don't think so. Thanks for pointing it out